shared/test/views_test.js

```diff
@@ -444,7 +444,6 @@ describe("loop.shared.views", function() {
       view = mountTestComponent({
         url: "fjrTykyw"
       });
-
```

Comment:
Please revert this; blank lines are used to separate the setup/execute/verify steps of a unit test.